assets/src/scripts/wdfn-home/store/wdfn-store.js

```diff
@@ -38,11 +38,11 @@ export const STORE_STRUCTURE = {
 const lookupSiteTypesFullNames = function (siteTypes) {
     // Translate keys into WQP understands
     const dictionary = {
-        groundwater: 'Aggregate groundwater use',
-        surfacewater: 'Aggregate surface-water-use',
+        groundwater: ['Aggregate groundwater use', 'Well', 'Subsurface'],
```

Comment:
This should've been its own commit, but I didn't realize until after pushing. This change just adds more search terms to each umbrella site type grouping.